sql/updates/latest-dev.sql

```diff
@@ -78,7 +78,7 @@ INSERT INTO _timescaledb_config.bgw_policy_drop_chunks_tmp (
       FROM _timescaledb_config.bgw_policy_drop_chunks
 );
 
-ALTER EXTENSION timescaledb DROP TABLE _timescaledb_config.bgw_policy_drop_chunks;
+---ALTER EXTENSION timescaledb DROP TABLE _timescaledb_config.bgw_policy_drop_chunks; ---why do we need this?
```

Comment:
```suggestion
ALTER EXTENSION timescaledb DROP TABLE _timescaledb_config.bgw_policy_drop_chunks;
```

It is needed because we re-create the table inside the extension, which means we need to drop it, but we cannot drop it if is is part of the extension, so we need to remove it from the extension.